sdk/storage/azdatalake/internal/generated/zz_models.go

```diff
@@ -52,7 +45,7 @@ type PathPrefix struct {
 // PathPropertiesInternal - Properties of a blob
 type PathPropertiesInternal struct {
 	// REQUIRED
-	ETag *azcore.ETag `xml:"Etag"`
+	Etag *string `xml:"Etag"`
```

Comment:
This type should be `azcore.ETag`.

Reply:
Thanks! Regenerated this after transform update.